ops/testing.py

```diff
@@ -1321,11 +1344,18 @@ def log_split(cls, message, max_len):
     def juju_log(self, level, msg):
         raise NotImplementedError(self.juju_log)
 
-    def get_pebble(self, socket_path: str):
+    def get_pebble(self, container: str, socket_path: str):
```

Comment:
the problem with having both is that it is entirely plausible to now have them disagree. (eg one thing called 'get_pebble(name, path)' and another did 'get_pebble(name, other_path)'. (yes, someone doing that is doing it wrong, but it *is* the api we are now exposing.
)

Reply:
This was reverted as noted above.  The one place in the test harness where we need the container name, I just parse it out of the socket path.  Not the greatest, but should work well enough.  We can always update that parsing if the socket path changes.